app/models/department.rb

```diff
@@ -0,0 +1,12 @@
+class Department < ApplicationRecord
+
+  validates_presence_of :name, :average_score
```

Comment:
premature to add the average score perhaps?

Reply:
We have removes traces of this future creep 👍🏽